vibe_code_apps_19/app/data_contract.py

```
@@ -211,29 +216,48 @@ def audit_quality_window(
+    """Return VAV id → AHU id from optional ``vav_to_ahu_simple.csv``.
+
+    ``parent_ahu`` is a first-class AHU column. Positional fallback is used only
+    when neither VAV nor AHU columns resolved — never overwrite a found
+    ``vav_id`` with column 0 (that mapped AHU ids onto tower numbers).
+    """
     path = Path(building_root) / "vav_to_ahu_simple.csv"
     if not path.is_file():
         return {}
     try:
         topo = pd.read_csv(path)
     except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)